App/Config/ReduxPersist.js

```diff
@@ -73,7 +73,7 @@ const REDUX_PERSIST = {
     // blacklist: ['nav', 'ipfs', 'auth', 'ui'],
     // Optionally, just specify the keys you DO want stored to persistence.
     // An empty array means 'don't store any reducers' -> infinitered/ignite#409
-    whitelist: ['preferences', 'uploadingImages', 'cameraRoll'],
+    whitelist: ['preferences', 'uploadingImages', 'cameraRoll', 'contacts'],
```

Comment:
Yea looks good for now. Doesn't seem like a good long term plan to rely on this persistence if this is the only place we're keeping track of contacts (like this should somehow be stored in a decentralized way?)